controllers/replace_failed_pods.go

```diff
@@ -52,29 +53,50 @@ func chooseNewRemovals(cluster *fdbtypes.FoundationDBCluster) bool {
 		return false
 	}
 
+	// The maximum number of removals will be the defined number in the cluster spec
+	// minus all currently ongoing removals e.g. process groups marked fro removal but
+	// not fully excluded.
+	removalCnt := 0
```

Comment:
Can we change this to `removalCount`? My brain consistently parses the word `cnt` as expanding to something else, so I would rather we spell it out 😊

Reply:
Sure :)